models/orm_base.py

```diff
@@ -1,10 +1,14 @@
 from sqlalchemy.orm import DeclarativeBase
 from sqlalchemy.inspection import inspect
+from sqlalchemy import LargeBinary
 
 
 class OrmBase(DeclarativeBase):
     def _asdict(self):
-        return {c.name: getattr(self, c.name) for c in inspect(type(self)).c}
+        return {
+            c.name: "<BLOB>" if isinstance(c.type, LargeBinary) else getattr(self, c.name)
+            for c in inspect(type(self)).c
+        }
```

Comment:
A minor optimization when printing objects, we don't want to print the contents of BLOBs.